src/main/java/Ui.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    private String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public Ui() {
+        
+    }
+
+    /**
+     * Greets user.
+     */
+    public String sayHi() {
+         return ("Hello I'm \n" + logo + "\nWhat can I do for you?");
+    }
+
+    /**
+     * Greets user and ends session.
+     */
+    public void sayBye() {
```

Comment:
It might be confusing that sayHi returns a String while sayBye does not return anything and simply prints a message. Perhaps sayBye might be better renamed to printGoodbyeMessage.